packages/pluggableWidgets/datagrid-web/src/Datagrid.xml

```diff
@@ -43,6 +43,19 @@
                                 <caption>Content</caption>
                                 <description/>
                             </property>
+                            <property key="width" type="enumeration" defaultValue="autoFill">
+                                <caption>Column width</caption>
+                                <description />
+                                <enumerationValues>
+                                    <enumerationValue key="autoFill">Auto-fill</enumerationValue>
```

Comment:
Tbh, it is not very self-explanatory label. Maybe `Fill available space` or something? Then also the key should be changed

Reply:
Same comment about the Layout grid here